test/eth-precompile/EthPrecompiles.js

```diff
@@ -0,0 +1,215 @@
+/*-
```

Comment:
Just for the consistency, can you please move this file into the native-precompiles/ folder? Just how you had the EthNativePrecompileCaller.sol in the contract/native-precompiles